gateways/node_api_gateway.py

```diff
@@ -122,21 +133,31 @@ def get_token_history(
             params["page"] = page
             params["timestamp"] = timestamp
 
-        return self.hathor_core_client.get(TOKEN_HISTORY_ENDPOINT, params=params)
+        return self.hathor_core_client.get(
+            TOKEN_HISTORY_ENDPOINT, params=params, timeout=NODE_API_TIMEOUT_IN_SECONDS
```

Comment:
Why don't we refactor this broken down to three lines, as the other functions?

Reply:
Oh! This is because of `black` code style.